appinventor/AICompanionApp.xcodeproj/project.pbxproj

```diff
@@ -352,6 +352,7 @@
 				TargetAttributes = {
 					06463EF71D95956800B23DCD = {
 						CreatedOnToolsVersion = 8.0;
+						DevelopmentTeam = A35W4MM59Y;
```

Comment:
I think all of these changes shouldn't have been added during github add, in a big project never do `git add .` . I know it's tedious but always do `git add <individual files you want to add>`

Reply:
Noted. I'll be more careful with git add in the future.